org.kde.kommit.json

```diff
@@ -0,0 +1,79 @@
+{
+    "id": "org.kde.kommit",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kommit",
+    "rename-icon": "kommit",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "modules": [
+        {
+            "name": "kommit",
```

Comment:
this should be last in the modules list.

Reply:
No, git is not a compile time dependency.

Reply:
doesn't matter. it's the convention.

Reply:
We still need this